source/installation/kafka-deployments/advanced-settings.txt

```diff
@@ -0,0 +1,164 @@
+.. _advanced-settings:
+
+=======================
+Advanced Kafka Settings
+=======================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+When you integrate Relational Migrator with Kafka, you can configure 
+additional advanced settings by updating your ``user.properties`` file.
+Your ``user.properties`` file is located in an OS specific location. For more 
+information, see :ref:`Relational Migrator File Locations <file-location>`.
+
+User Properties
+---------------
+
+Provide optional configuration settings by updating your ``user.properties`` file. 
+
+.. glossary::
+
+   ``spring.profiles.active``
+    Default: local
+
+    Relational Migrator can be run in the following profile: 
+
+    - ``kafka``: Remote server using embedded server deployment.
+    - ``confluent``: Remote server using Confluent Cloud for managed Kafka.
+
+   ``server.port``
+    Default: ``8278``
+
+    The port the Relational Migrator application runs on.
+
+   ``migrator.kafka.bootstrap.servers``
+    This is a list of ``host:port`` pairs of your pre-existing kafka cluster. For 
+    more information, see `Worker Configuration Properties 
+    <https://docs.confluent.io/platform/current/connect/userguide.html#worker-configuration-properties-file>`__. 
+
+   ``migrator.kafka.connect.url``
+    The Kafka Connect host url using the ``{host}:{port}`` format. 
+
+   ``migrator.kafka.connect.metrics.jmx.service.url``
+    The Kafka Connect Java Management Extensions (JMX) url. 
+
+   ``migrator.kafka.connect.metrics.jmx.ssl``
+    Default: false
+    
+    Set to ``true`` if SSL is enabled.
+
+    You can add additonal remote Java Management Extensions (JMX) for SSL 
+    configuration by prefixing them with ``migrator.kafka.connect.metrics.jmx.properties``.
+
+   ``migrator.kafka.enable.topic.cleanup``
+    Default: false
+    
+    Cleans up the topics created during the last migration job.
+
+   ``migrator.connector.source.common.errors.max.retries``
+    Default: 5
```

Comment:
Monospace default number value for consistency.

Reply:
Thanks for bringing this up! I am actually going to follow the server parameters page example and not use monospacing for the values and italicize the _Default_.